plugins/woocommerce/includes/admin/settings/class-wc-settings-emails.php

```diff
@@ -134,6 +152,31 @@ protected function get_settings_for_default_section() {
 				),
 			);
 
+			$additional_fonts = array();
+			if ( wc_current_theme_is_fse_theme() && class_exists( 'WP_Font_Face_Resolver' ) ) {
+				$theme_fonts = WP_Font_Face_Resolver::get_fonts_from_theme_json();
+				if ( count( $theme_fonts ) > 0 ) {
+					foreach ( $theme_fonts as $font ) {
+						if ( ! empty( $font[0]['font-family'] ) ) {
+							$additional_fonts[ $font[0]['font-family'] ] = $font[0]['font-family'];
+						}
+					}
+					ksort( $additional_fonts );
+				}
+			}
+			$font_family = array(
+				'title'    => __( 'Font family', 'woocommerce' ),
+				'id'       => 'woocommerce_email_font_family',
+				'desc_tip' => '',
+				'default'  => 'Arial',
+				'type'     => 'select',
+				'class'    => 'wc-enhanced-select',
+				'options'  => array_merge(
+					array_combine( array_keys( self::$font ), array_keys( self::$font ) ),
+					$additional_fonts
+				),
```

Comment:
What do you think about `ksort()`ing this array? From the user's perspective, even if the beginning is sorted (the email-safe fonts), the end of the list of fonts will discard that assumption once theme fonts are appended.

Reply:
We can do that for sure. Maybe a better way to separate standard and custom fonts would be to introduce optgroups like in MailPoet. What do you think about that?

Reply:
That's a good idea, I like it. I'll let you decide which one to pick.

Reply:
The groups are a nice addition 👍 Just a small note, that the `Custom fonts` is empty when no custom fonts are available, which looks broken (as in the fonts haven't loaded). I'd probably just hide the group completely.

Reply:
Nice catch! Fixed.